lib/plugins/aws/lib/monitorStack.test.js

```diff
@@ -150,18 +180,32 @@ describe('monitorStack', () => {
         StackEvents: [
           {
             EventId: '1a2b3c4d',
-            LogicalResourceId: 'mocha',
+            StackName: 'new-service-dev',
+            LogicalResourceId: 'new-service-dev',
             ResourceType: 'AWS::CloudFormation::Stack',
             Timestamp: new Date(),
             ResourceStatus: 'DELETE_IN_PROGRESS',
           },
         ],
       };
+      const nestedStackEvent = {
```

Comment:
This should be removed here and one extra unit test should be added which covers this scenario.